scripts/build_utils.sh

```diff
@@ -838,17 +836,14 @@ ceph_build_args_from_flavor_and_dist() {
         # - we need to test old clients which are built on bionic. for instance, we don't build
         #   nautilus on focal yet.
         # - some upgrade tests still include bionic facets.
-        if [ "$dist" == "bionic" ]; then
-            local ver
-            ver=$(git describe --abbrev=8 --match "v*" | sed s/^v// | cut -f1 -d'.')
-            # use libc allocator when building on quincy and up on bionic
-            if [ ${ver} -lt 17 ]; then
-                CEPH_EXTRA_CMAKE_ARGS+=" -DALLOCATOR=tcmalloc"
-            fi
-        else
-            # always use tcmalloc on non-bionic dists
-            CEPH_EXTRA_CMAKE_ARGS+=" -DALLOCATOR=tcmalloc"
-        fi
+        #
+        # do not specify the allocator and let cmake figure it out:
+        # - on quincy and up, gperftools 2.6.2 is required, so we are not impacted by
+        #   https://tracker.ceph.com/issues/49139. and cmake uses tcmalloc if gperftools 2.6.2 and
```

Comment:
I think you meant https://tracker.ceph.com/issues/39703 here.

Reply:
yes, sorry. was looking at the wrong browser tab.